### DISCUSSION TITLE
可手动删除Java

### DISCUSSION THREAD

**Discussion by 12234254** [Meloong-Git/PCL#253] [提问] [answered]
![image](https://user-images.githubusercontent.com/103358922/196679699-9fc2826e-b8db-4ae5-95b6-79e85008e359.png)

**Comment by Pigeon0v0** [comment]
自动搜索一下应该就没了

**Comment by LTCatt** [comment, marked as answer]
你可以自己去那个文件夹下面把 java 删了……这不是启动器应该干的事，也容易误删重要文件。

顺带一提， @bwtx2022 不要把自己 mark as answer……有的明显你就理解错了喂（

**Reply by Pigeon0v0** [reply]
行（
